cmd/thanos/rule.go

```diff
@@ -604,6 +598,8 @@ func runRule(
 		}, func(error) {
 			cancel()
 		})
+	} else {
+		level.Info(logger).Log("msg", "No supported bucket was configured, uploads will be disabled")
```

Comment:
nice, can we also fix the sentence one we are here?  (should be lower case all)
```suggestion
		level.Info(logger).Log("msg", "no supported bucket was configured, uploads will be disabled")
```